quesma/quesma/schema_transformer_test.go

```diff
@@ -663,3 +664,313 @@ func TestApplyWildCard(t *testing.T) {
 		})
 	}
 }
+
+func TestApplyPhysicalFromExpression(t *testing.T) {
+
+	indexConfig := map[string]config.IndexConfiguration{
+		"test": {
+			Name:    "kibana_sample_data_ecommerce",
+			Enabled: true,
```

Comment:
You will have to remove it after https://github.com/QuesmaOrg/quesma/pull/682

Reply:
++ please rebase 😉

Reply:
I'll remove.